app/assets/javascripts/backbone/views/calendar_view.js.coffee

```diff
@@ -0,0 +1,21 @@
+class MS.Views.CalendarView extends Backbone.View
+
+  el: 'body'
```

Comment:
el should be `#calendar` perhaps?

Try to stay away from having `body` as the `el` unless your view is really responsible for the entire page. This view is only responsible for the calendar.

Then, below, you can replace `$('#calendar')` with `@$el`.

We talked about `@$` and `@$el` the other day. If you don't quite understand them, please come talk to me.